CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to the "prettier-vscode" extension will be documented in thi
 
 <!-- Check [Keep a Changelog](https://keepachangelog.com/) for recommendations on how to structure this file. -->
 
+## [9.12.0]
```

Comment:
This should be a major release. 10.0.0

Reply:
Maybe we should release v10 after Prettier releases v3? This PR is not breaking anything, just add support for v3 alpha.

Reply:
@ntotten I agree with fisker. If you too can agree with this, please merge and release.

Reply:
Seems there is a release work flow. So I guess we can create a `v9.12.0` tag to trigger release? https://github.com/prettier/prettier-vscode/blob/8bd9176a791c895bf38604acfb102c87375fc515/.github/workflows/main.yml#L73

But let's wait for ntotten to respond first.